rep-2004.rst

```diff
@@ -434,86 +354,318 @@ For example, this might be a tutorial package which is not intended for reuse bu
 Package Requirements
 ~~~~~~~~~~~~~~~~~~~~
 
-*Note: bullets below that start with [ROS Core], will be the prescription for what we do in the core packages in order to meet the associated requirements*
-
 Requirements to be considered a 'Level 4' package:
 
-* Version Policy:
-
-  * No requirements, but having a policy is still recommended (e.g. ``semver``), even if the version is not yet stable (e.g. >= 1.0.0 for ``semver``)
+#. **Version Policy:**
 
-* Change Control Process:
+   #. No requirements, but having a policy is still recommended (e.g. ``semver``), even if the version is not yet stable (e.g. >= 1.0.0 for ``semver``)
 
-  * No explicit change control process required, but still recommended
+#. **Change Control Process:**
 
-* Documentation:
+   #. No explicit change control process required, but still recommended
 
-  * Must have a declared license or set of licenses
-  * Must have a copyright statement in each source file
-  * [ROS Core]:
+#. **Documentation:**
 
-    * Must have automated checks for copyright statements and licenses
-    * Must use the Apache 2.0 license, unless the package has an existing permissive license (e.g. rviz uses three-clause BSD)
+   #. Must have a declared license or set of licenses
+   #. Must have a copyright statement in each source file
 
-* Testing:
+#. **Testing:**
 
-  * No explicit testing requirements, though covering some if not all of the features with tests is recommended
+   #. No explicit testing requirements, though covering some if not all of the features with tests is recommended
 
-* Dependencies:
+#. **Dependencies:**
 
-  * No restrictions
+   #. No restrictions
 
-* Platform Support:
+#. **Platform Support:**
 
-  * May support all tier 1 platforms for ROS 2, as defined in `REP-2000 <https://www.ros.org/reps/rep-2000.html#support-tiers>`_
+   #. May support all tier 1 platforms for ROS 2, as defined in `REP-2000 <https://www.ros.org/reps/rep-2000.html#support-tiers>`_
 
 Any package that does not claim to be 'Level 3' or higher is automatically 'Level 4'.
-Refer to the detailed description of the requirements in the Quality Level 1 section above for more information.
+Refer to the detailed description of the requirements following the Quality Level 1 section above for more information.
 
 Quality Level 5
 ^^^^^^^^^^^^^^^
 
-Packages in this category simply do not meet even the 'Level 4' requirements, and for that reason should not be used.
-The rationale being that all packages should have at least a declare license or licenses and should include copyright statements in each file.
+Packages in this category cannot even meet the simple 'Level 4' requirements, and for that reason should not be used.
+All packages should have at least a declared license or set of licenses and should include copyright statements in each file.
+
+Quality Level Comparison Chart
```

Comment:
one reason leaving all the written sections with the chart doesn't work is that the auto numbering makes it so the requirements' numbers don't all match level 1. for example, level 1's `2.iv` is `Must have Continuous Integration (CI) policy for all change requests` but for level 2 the same requirement is listed as `2.ii` because it doesn't have all the other requirements level 1 does. this makes the numbered rows in the chart slightly ambiguous because `4.i` means different things for different levels.

this can be fixed by manually numbering requirements in the lists for 2+ to match level 1, or rearranging the requirements in level 1 so they match the order in which they're deprecated in the following levels. 

didn't do any of this yet because we don't know what we're doing with the chart yet.